### PR TITLE
Feature use dev awesome font

### LINKED ISSUE #430: Use Gobstones and Wollok SVGs

### PR DESCRIPTION
**Pull request by fedescarpa**
fixes #430

### CODE REVIEW

**Comment by fedescarpa** [comment]
Mumuki logo in markdown
da-mumuki-markdown

Mumuki logo in console prompt
mumuki prompt

**Comment by flbulgarelli** [comment]
You rock!

File: app/assets/javascripts/application/console.js [hunk 1]
```diff
@@ -1,16 +1,26 @@
 var mumuki = mumuki || {};
 (function (mumuki) {
+  function renderPrompt() {
+    if ($('#prompt').attr('value').indexOf('ム') >= 0) {
+      $('.jquery-console-prompt-label')
+        .html('')
+        .append('<i class="text-primary da da-mumuki"></i>')
```
Comment (flbulgarelli) [inline app/assets/javascripts/application/console.js]: Jaja great hack

Comment (fedescarpa) [inline app/assets/javascripts/application/console.js]: :sunglasses:

**Pull request merged**